### ISSUE TITLE
Dragging Preview of Component inside Cell in Safari

### ISSUE THREAD

**Issue by BugsBunny338** [facebook/fixed-data-table#372]
Not sure whether this is an issue of `fixed-data-table`, [`react-dnd`](https://github.com/gaearon/react-dnd) or Safari, but when I insert a draggable ([`export default DragSource(type, spec, collect)(MyComponent);`](http://gaearon.github.io/react-dnd/docs-drag-source.html)) component inside a `Cell` (inside `Column` inside `Table`) and start dragging, there is no preview in Safari 9 on OS X 10.11. Dropping on defined dropzones work, but no thumbnail midflight :-(

If I use the same [`DragSource`](http://gaearon.github.io/react-dnd/docs-drag-source.html) component outside `fixed-data-table`, dragging preview works normally.

Any ideas?

**Comment by cyranosky** [comment]
Hi,can i ask you a question?  fixed-data-table  can support drag and drop?

**Comment by BugsBunny338** [comment]
@cyranosky No, it does not. That's why I use Abramov's [`react-dnd`](https://github.com/react-dnd/react-dnd).
